Web/scripts/fluid_properties.IF97Profile.py

```
@@ -0,0 +1,194 @@
+    if97 = AbstractState('IF97', 'Water')
+    hs, prs = [], []
+    for T in Ts:
+        for p in ps:
+            try:
+                if97.update(CP.PT_INPUTS, p, T)
+                h = if97.hmass()
+                if np.isfinite(h):
+                    hs.append(h)
+                    prs.append(p)
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)